.github/workflows/test_suite.yml

```diff
@@ -0,0 +1,68 @@
+name: 🕵️ Test suite
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Run every day at 23:59 UTC
+    - cron: 59 23 * * *
```

Comment:
@digitronik why have a schedule here for the actions?

Reply:
@mshriver In the absence of PRs, this will help to verify the installation doesn't have any dependency issue so I decided to run at least one time in a day.